store/postgres/src/deployment.rs

```diff
@@ -748,10 +748,7 @@ fn insert_subgraph_error(conn: &PgConnection, error: &SubgraphError) -> anyhow::
     } = error;
 
     let block_num = match &block_ptr {
-        None => {
-            assert_eq!(*deterministic, false);
```

Comment:
Why was this assertion removed?

Reply:
This was discussed with the team, as far as I can understand this was a sanity check that determisnistic errors could only happen with a block number being set, this is not longer the case with substreams, since the deterministic error can even happen at boot time, for example, with incorrect number of arguments.